config/prod/JKG-SciComp-S3.yaml

```diff
@@ -1,6 +1,13 @@
 # Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
-template_path: "remote/s3-bucket.yaml"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/s3-bucket.yaml"
 stack_name: "JKG-SciComp-S3"
+stack_tags:
+  Department: "SysBio"
+  Project: "amp-ad"
+  OwnerEmail: 'jake.gockley@sagebase.org'
+  CostCenter: "NIA AMP-AD CC / 101500"
```

Comment:
Added @jgockley62  as a reviewer, to confirm this CostCenter.

Reply:
@brucehoff not familiar with the new cost center tags, but this is for AMP-AD

Reply:
@jgockley62 If you haven't been provided a specific cost center than the next step would be for you to ask the project lead/PI what program funds the work. Finance maintains a list of six digit program codes, and the project leads/PIs should know where their funding is coming from.

Reply:
@zaro0508 Since Jake has not suggested a different code, I reviewed the available codes and I think this one ("NIA AMP-AD CC / 101500") is the best we can guess.